README.md

```diff
@@ -234,7 +237,7 @@ namespace TSQLLint.Tests.UnitTests.PluginHandler
         public RuleViolationSeverity Severity { get; private set; }
         public string Text { get; private set; }
 
-        public TestRuleViolation(string fileName, string ruleName, string text, int lineNumber, int column, RuleViolationSeverity ruleViolationSeverity)
+        public SampleRuleViolation(string fileName, string ruleName, string text, int lineNumber, int column, RuleViolationSeverity ruleViolationSeverity)

```

Comment:
This simply isn't accurate currently.  The constructor for a class must have the same name as the class.